templates/zerver/help/create-a-stream.md

```diff
@@ -12,29 +15,45 @@ Streams can be either:
 
     ![Streams Overview](/static/images/help/streams-overview.png)
```

Comment:
You can probably delete this image reference (but not the file itself) since we already showed the same exact screenshot in the previous step.